packages/mdx/test/compile.js

```diff
@@ -858,13 +858,11 @@ test('jsx', async () => {
     [
       '/*@jsxRuntime automatic @jsxImportSource react*/',
       'function _createMdxContent(props) {',
-      '  const _components = Object.assign({',
-      '    "a-b": "a-b"',
-      '  }, props.components);',
-      '  return <>{<_components.a-b></_components.a-b>}</>;',
+      '  const _components = props.components || ({}), _component0 = "a-b";',
```

Comment:
We should still take this from `components['a-b']` though, so people can pass them in? 🤔 Or am I missing something and is it intentional that you’re not doing that?